### ISSUE TITLE
False positive: Unused method parameter "args: Array[String]" on "main"

### ISSUE THREAD

**Issue by RichardBradley** [scapegoat-scala/scapegoat#105]
Should `Array[String]` arguments to "main" methods get excluded from UnusedMethodParameter by default? 

It's much less suspicious than most ignored parameters IMO.

**Comment by RichardBradley** [comment]
I notice that "SuppressWarnings" doesn't work per-param, like it does in Java / Eclipse:

```
  def main(args: Array[String] @SuppressWarnings(Array("UnusedMethodParameter"))): Unit = {
   ...
```

**Comment by sksamuel** [comment]
No its method level only, and good find on the main method.

**Comment by hzwaal** [comment]
As a workaround you could extend from `scala.App`, although this may not always be what you want.

**Comment by saeltz** [comment]
Fixed by #203.
